### ISSUE TITLE
Fix header return link focus style

### ISSUE THREAD

**Issue by schwartzadev** [COVID19Tracking/website#1267]
![image](https://user-images.githubusercontent.com/18607205/87574169-3dee4280-c68b-11ea-8978-90818729c96c.png)

This focus outline and color should be white. Let's do:

- [x] `@include link-hover-dark-bg` for hover
- [x] `@include link-focus-dark-bg` for focus
